app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -178,11 +178,13 @@ protected void setSpoilerAndContent(boolean expanded,
                                         @Nullable String spoilerText,
                                         @Nullable Status.Mention[] mentions,
                                         @NonNull List<Emoji> emojis,
+                                        @Nullable PollViewData poll,
                                         final StatusActionListener listener) {
+        setupPoll(poll, emojis, listener);
```

Comment:
This is called even when the poll is currently not visible, right? Why not move it somewhere where its only executed when the poll is actually visible, e.g. into `setTextVisible`?

Reply:
I just kept the old code flow as much as possible - I can make this change if you prefer

Reply:
Should be a quick change, so yes please do it

Reply:
Updated